src/Raven.Server/Smuggler/Documents/MultiShardedDestination.cs

```diff
@@ -44,6 +44,7 @@ public MultiShardedDestination([NotNull] ISmugglerSource source, [NotNull] Shard
 
         public async ValueTask<IAsyncDisposable> InitializeAsync(DatabaseSmugglerOptionsServerSide options, SmugglerResult result, long buildVersion)
         {
+            options.OperateOnDatabaseRecordTypes &= ~(DatabaseRecordItemType.HubPullReplications | DatabaseRecordItemType.SinkPullReplications);
```

Comment:
Better to move those flags to DatabaseSmugglerOptions as internal and call them `ShardingNotSupportedDatabaseSmugglerOptions` or sth like that.

Reply:
Done